app/Http/Controllers/UserController.php

```diff
@@ -116,13 +128,22 @@ public function destroy(User $user)
     {
         try {
             if ($user->id == Auth::user()->id) {
-                return redirect()->back()->with('messageFail', 'Delete failed, Cannot delete myself');
+                return redirect()->back()->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'Delete failed, Cannot delete myself'
+                ]);
             } else {
                 $this->userRepository->delete($user->id);
-                return redirect('/admin/users')->with('messageSuccess', 'This user successfully deleted');
+                return redirect('/admin/users')->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This user successfully deleted'
+                ]);
             }
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/UserController.php</br>**Line**: 145</br>Each line in an array declaration must end in a comma